### PR TITLE
[Task#79954] Make category management for admin

### PR DESCRIPTION
**Pull request by mvansan**
https://edu-redmine.sun-asterisk.vn/issues/79954

### CODE REVIEW

**Comment by mvansan** [comment]
ready

**Comment by ghost** [comment]
đặt lại title 
[Task#79954] Make category management for admin

**Review by minhsieu9102003 — APPROVED** [review]

File: ecommerce-app/app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    public function create()
+    {
+        return view('admin.categories.create');
+    }
+
+    public function store(Request $request)
+    {
+        $request->validate([
```
Comment (ghost) [inline ecommerce-app/app/Http/Controllers/CategoryController.php]: sử dụng https://laravel.com/docs/11.x/validation#form-request-validation

File: ecommerce-app/app/Http/Controllers/CategoryController.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    {
+        $category = ProductCategory::findOrFail($id);
+        return view('admin.categories.edit',compact('category'));
+    }
+
+    public function update(Request $request, $id)
+    {
+        $category = ProductCategory::findOrFail($id);
+
+        $request->validate([
```
Comment (ghost) [inline ecommerce-app/app/Http/Controllers/CategoryController.php]: https://laravel.com/docs/11.x/validation#form-request-validation

**Review by ghost — APPROVED** [review]

**Review by notverygood135 — APPROVED** [review]

**Review by sonnguyendev157 — APPROVED** [review]

**Review by tuan-bach123 — APPROVED** [review]

**Pull request merged**